third_party/libzmq.BUILD

```diff
@@ -0,0 +1,241 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # LGPL license with exception for static linking (for libzmq)
+
+genrule(
+    name = "configure",
+    outs = ["platform.hpp"],
+    cmd = "cd external/libzmq && ./configure && cd ../..; cp external/libzmq/src/platform.hpp $@",
+)
```

Comment:
Not sure what the best thing to do here is. Do you build on windows using MSVC? The [`configure`](https://github.com/zeromq/libzmq/blob/3f80657c247bf14ba29d6de3cf8aeea8d0f1bc87/configure.ac) script of libzmq seems pretty sophisticated and I don't know whether just copying the file is sufficient. I'm afraid I don't have a windows machine to test it.